helm/openwhisk/runtimes.json

```diff
@@ -166,6 +166,20 @@
                     "attachmentName": "codefile",
                     "attachmentType": "text/plain"
                 }
+            },
```

Comment:
change default above to false.
I wonder if this should be flagged as a bug in the controller when more than one kind is set as default.